eventhubs-capture-synapse-delta/README.md

```diff
@@ -0,0 +1,202 @@
+The provided scripts will deploy an end-to-end solution complete with load test client.
+
+## Running the Scripts
+
+Please note that the scripts have been tested on [Ubuntu 18 LTS](http://releases.ubuntu.com/18.04/), so make sure to use that environment to run the scripts. You can run it using Docker, WSL or a VM:
+
+- [Ubuntu Docker Image](https://hub.docker.com/_/ubuntu/)
+- [WSL Ubuntu 18.04 LTS](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab)
+- [Ubuntu 18.04 LTS Azure VM](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuServer1804LTS)
+
+The following tools/languages are also needed:
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-apt?view=azure-cli-latest)
+  - Install: `sudo apt install azure-cli`
+- [jq](https://stedolan.github.io/jq/download/)
+  - Install: `sudo apt install jq`
+- [python](https://www.python.org/)
+  - Install: `sudo apt install python python-pip`
+
+## Setup Solution
+
+Make sure you are logged into your Azure account:
+
+    az login
+
+and also make sure you have the subscription you want to use selected
+
+    az account list
+
+if you want to select a specific subscription use the following command
+
+    az account set --subscription <subscription_name>
+
+once you have selected the subscription you want to use just execute the following command
+
+    ./create-solution.sh -d <solution_name> -p <synapse_sql_password> -w <wait_verfication>
+
+- The `solution_name` value is used to create a resource group that contains all resources created by the script. It is also be used as a prefix for all resource create so, in order to help to avoid name duplicates that will break the script, you may want to generate a name using a unique prefix. **Please also use only lowercase letters and numbers only**, since the `solution_name` is also used to create a storage account, which has several constraints on characters usage:
+
+[Storage Naming Conventions and Limits](https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions#storage)
+
+- The `sql-admin-login-password` value is be used to create the password for the synapse sql pool. The SQL Pools are not explicitly used anywhere in the project, but the SQL Admin Login Password is necessary when creating a Synapse Workspace.
+
+- `wait_verification` is a boolean flag defaulted to `true` that places the CLI in a waiting state until the pipeline results have to downloaded and written to SDTOUT.
+
+To have an overview of all the supported arguments just run
+
+    ./create-solution.sh
+
+**Note**
+To make sure that name collisions will be unlikely, you should use a random string to give name to your solution. The following script will generated a 7 random lowercase letter name for you:
+
+    ./_common/generate-solution-name.sh
+
+## Running the Steps
```

Comment:
I don't see this section in any of the other README files, even though (I believe) they implement similar steps.